src/components/admin/filterNumberRange.js

```diff
@@ -0,0 +1,24 @@
+adminApp.filterNumberRange = {
+  view: function(ctrl, args){
+    return m('.w-col.w-col-3.w-col-small-6', [
+            m('label.fontsize-smaller[for="field-6"]', args.label),
```

Comment:
Just used the index as we have full control of forms been generated here. But definitely we could work out a way to build an identifier based on arguments.